Automattic/simplenote-ios
# Interlinks Autocomplete: Mark IV

### Fix
In this PR we're:

1. Fixing the Blur's behavior
2. Wiring **SPShadowView**
3. Updating **SPShadowView** so that it's _IBDesignable™_
4. Dropping the last Autocomplete row's separator

cc @eshurakov 
Ref. #914

### Test: Style
1. Open any note
2. Type `[keyword` with `keyword` being a word present in any of your note's titles

- [x] Verify the Autocomplete UI has rounded corners
- [x] Verify the shadow looks great!
- [ ] Verify the Autocomplete has a (slightly) blurry background!

### Test: Regressions
- [x] Verify the Note History still looks great
- [x] Verify the Note Info also looks super!

### Release
> `RELEASE-NOTES.txt` was updated in d3adb3ef with:
> 
> > Added markdown support
